build.gradle

```diff
@@ -202,7 +202,7 @@ sonarqube {
     property "sonar.pitest.reportsDirectory", "build/reports/pitest"
     // FIXME : RDM-11309
     property "sonar.exclusions",  "**/service/impl/DocumentManagementServiceImpl.java,"
-    property "sonar.coverage.exclusions", "**/configuration/*.java,**/advice/exception/*.java,**/apihelper/*.java,**/ErrorConstants.java"
+    property "sonar.coverage.exclusions", "**/configuration/*.java,**/advice/exception/*.java,**/apihelper/*.java,**/ErrorConstants.java,**/EnumValueValidator.java"
```

Comment:
you can remove this if not needed or i guess class name has changed to ClassificationValidator right?